packages/xpetra/src/Headers/Xpetra_UseShortNamesScalar.hpp

```diff
@@ -71,6 +71,7 @@ typedef Xpetra::BlockedVector<Scalar, LocalOrdinal, GlobalOrdinal, Node> Blocked
 
 #ifdef XPETRA_MULTIVECTOR_SHORT
 typedef Xpetra::MultiVector<Scalar, LocalOrdinal, GlobalOrdinal, Node> MultiVector;
+typedef Xpetra::MultiVector<double, LocalOrdinal, GlobalOrdinal, Node> MultiVectorReal;
```

Comment:
I don't like the name. In fact, what you want is a non-complex scalar. Furthermore the "real" should not be an appendix but prepended to be consistent with the special typedefs that we already have for Vectors with scalar = int.
I'd strongly recommend a name like RealValuedMultiVector instead of MultiVectorReal. This would be in compliance with the other typedefs that we already have: LocalOrdinalMultiVector, LOMultiVector, and so on.

Reply:
Please move the typedefs in Xpetra_UseShortNamesOrdinal.hpp. Or, alternatively, move the typedefs for LOMultiVector etc. to this file.
Please also add the corresponding typedefs for Vector (even if they are not used now).